rllib/BUILD

```diff
@@ -1,16 +1,19 @@
 # --------------------------------------------------------------------
 # BAZEL/Travis-ci test cases.
-#
-# NOTE: Move more test cases here from ci/jenkins/run_rllib_tests.sh
-# as Travis tests seem to run more stable.
 # --------------------------------------------------------------------
 
-
 # ---------------------------------------
-# Agents (short learning tasks)
+# Agents learning regression tests
 # ---------------------------------------
 
-# TODO:
+py_test(
+    name = "tests/run_regression_tests.py",
```

Comment:
Name shouldn't end with .py (bazel will complain)

Reply:
Probably can't have slashes either.

Reply:
Yup, that's true, fixed that.
Should be all good now. Just waiting for tests to pass (looks good locally).